scholly/base/src/components/App.js

```diff
@@ -3,6 +3,8 @@
  */
 import React from 'react';
 import { Switch, Route } from 'react-router-dom';
+import { Helmet } from 'react-helmet';
```

Comment:
Nice never new this existed. It makes updating the title much simpler than I had initially thought.